search_action_domain/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Search Action Domain
+====================
+
+This module lets the user to use server actions as a filters in the "Favorites"
+section, in other words, the user might use complex queries to search the records.  
+
+To use this kind of filter, go to:
+
+1. Settings > Technical > Actions > Server Actions > Create
+
+2. Select model and set the state as a Python code
+
+3. Write a valid Python code to search in the selected model
+
+4. Press Action > Create As Filter
+
+5. Finally, find the created filter in "Favorites" section in the selected model
+
+Server Action:
+
+    .. image:: ../search_action_domain/static/description/search_action_domain_action.png
+       :alt: search_action_domain: Create As Filter
+       :width: 400px
+
+Filter:
+
+    .. image:: ../search_action_domain/static/description/search_action_domain_filter.png
```

Comment:
Are you sure about ".." in the path?

Check if that works functionally (opening module description).

Reply:
Yes, it is working, actually, I copied it from account_chatter module